### PR TITLE
Reactify course view - DO NOT CHECK IN

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: code-studio/src/js/components/EnrollmentStatus.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+          <div className={ribbonClass}/>
+        </div>
+      </div>
+    );
+  }
+});
+
+var CourseUnitSection = React.createClass({
+  propTypes: {
+    courseUnit: React.PropTypes.object
```
Comment (Bjvanminnen) [inline code-studio/src/js/components/EnrollmentStatus.jsx:26]: Could use `React.PropTypes.shape` to both document and validate what you expect this object to look like.

Comment (Bjvanminnen) [inline code-studio/src/js/components/EnrollmentStatus.jsx:26]: You can also append `.isRequired` to your PropTypes in cases where they should always be there.

File: code-studio/src/css/plc/teacher_dashboard.scss [hunk 2]
```diff
@@ -64,25 +64,48 @@
...
+        overflow: hidden;
+        position: absolute;
+        top: 0px;
+        right: 0px;
+
+        .ribbon {
+          -webkit-transform: rotate(45deg);
+          -moz-transform:    rotate(45deg);
+          -ms-transform:     rotate(45deg);
+          -o-transform:      rotate(45deg);
```
Comment (joshlory) [inline code-studio/src/css/plc/teacher_dashboard.scss:84]: I see vendor-prefixed `transform` but not un-prefixed.  I believe we have a SCSS mixin that will generate the vendor prefixes.

File: code-studio/src/css/plc/teacher_dashboard.scss [hunk 3]
```diff
@@ -64,25 +64,48 @@
       margin: 10px;
       padding: 5px;
 
+      background-color: #0094CA;
```
Comment (joshlory) [inline code-studio/src/css/plc/teacher_dashboard.scss:67]: Should this be a SCSS constant?

**Pull request closed without merging**